Modules/Libs/QuestieLib.lua

```diff
@@ -535,3 +535,114 @@ function QuestieLib.tunpack(tbl)
 
     return recursion(1)
 end
+
+
+local frameObject = nil
+if _G["QuestLogObjectivesText"] then -- classic
+    frameObject = _G["QuestLogObjectivesText"] 
+elseif _G["QuestInfoObjectivesText"] then -- Wotlk Classic
+    frameObject = _G["QuestInfoObjectivesText"]
+end
+--Part of the GameTooltipWrapDescription function
+local objectiveFontString = UIParent:CreateFontString("questieObjectiveTextString", "ARTWORK", "QuestFont")
+objectiveFontString:SetWidth(frameObject:GetWidth() or 275) --QuestLogObjectivesText default width = 275
+objectiveFontString:SetHeight(0);
+objectiveFontString:SetPoint("LEFT");
+objectiveFontString:SetJustifyH("LEFT");
+---@diagnostic disable-next-line: redundant-parameter
+objectiveFontString:SetWordWrap(true)
+objectiveFontString:SetVertexColor(1,1,1, 1)--Set opacity to 0, even if it is shown it should be invisible
+local font, size = frameObject:GetFont()
+--Chinese? "Fonts\\ARKai_T.ttf"
+objectiveFontString:SetFont(font, size);
+objectiveFontString:Hide()
+
+---Emulates the wrapping of a quest description
+---@param line string @The line to wrap
+---@param prefix string @The prefix to add to the line
+---@param combineTrailing boolean @If the last line is only one word, combine it with previous? TRUE=COMBINE, FALSE=NOT COMBINE, default: true
+---@param splitOnDot boolean @Should we add a linebreak if a dot appears thats not at the end of a line TRUE=NEW ROW, FALSE=NO NEW ROW, default: true
```

Comment:
Do we need both parameters if they are true in both usages?

Reply:
Well, i made it general so that we can control these things in other places if we want. While they default to true maybe one would want to not have these behaviors?

Reply:
Sounds fine to me 👍🏻